src/main/java/com/streisky/virtualshowcasebackend/controller/product/ProductController.java

```diff
@@ -31,61 +30,37 @@ public class ProductController {
 
     @PostMapping
     @Transactional
-    public ResponseEntity<ProductDTO> register(@RequestBody @Valid ProductDTO productDTO) {
-        try {
-            ProductDTO savedProduct = productService.save(productDTO);
-            URI location = ServletUriComponentsBuilder
-                    .fromCurrentRequest()
-                    .path("/{id}")
-                    .buildAndExpand(savedProduct.id())
-                    .toUri();
+    public ResponseEntity<ProductDTO> save(@RequestBody @Valid ProductDTO productDTO) {
+        ProductDTO savedProduct = productService.save(productDTO);
+        URI location = ServletUriComponentsBuilder
+                .fromCurrentRequest()
```

Comment:
Oh that is nice! Does it successfully gets the current endpoint path?

Reply:
Yeah!

Returns the GET path for query in the headers.